ocaml/idl/datamodel.ml

```diff
@@ -2214,6 +2214,15 @@ module Tunnel = struct
           ; param_release= next_release
           ; param_default= Some (VEnum "gre")
           }
+        ; {
+            param_type= Bool
+          ; param_name= "cross_server"
+          ; param_doc=
+              "Different scenario used for the tunnel \
```

Comment:
The documentation does not explain the purpose and consequence of this value. What does it mean when this value is true?